### DISCUSSION TITLE
Welcome to Application-web Discussions!

### DISCUSSION THREAD

**Discussion by AgroOnline** [AgroOnline-Dev/Application-web#27] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AgroOnline** [comment]
Hello all web developers, @shakur2209 and @LoicFalla 
This is our space where we can discuss and exchange... if anybody is having a problem here he can just tag me and I will quickly reply

**Reply by aleck2209** [reply]
@Noumbissi16 okay
